src/gui/creds/oauth.h

```diff
@@ -27,15 +27,20 @@ namespace OCC {
  *
  *   --> start()
  *       |
- *       +----> openBrowser() open the browser to the login page, redirects to http://localhost:xxx
+ *       +----> fetchWellKnown() query the ".well-known/openid-configuration" endpoint
```

Comment:
Please also document what happens if this does does not exist and how this is with regards to OpenID vs normal OAuth?